tests/test_ingest.py

```diff
@@ -0,0 +1,191 @@
+# This file is part of obs_subaru.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Unit tests for Gen3 HSC raw data ingest.
+"""
+
+import unittest
+import tempfile
+import os
+import shutil
+import lsst.utils.tests
+
+from lsst.afw.image import LOCAL
+from lsst.afw.geom import Box2I, Point2I
+from lsst.daf.butler import Butler
+from lsst.obs.hsc import HscMapper
+from lsst.obs.subaru.gen3.hsc import HyperSuprimeCamRawIngestTask, HyperSuprimeCam
+
+testDataPackage = "testdata_subaru"
+try:
+    testDataDirectory = lsst.utils.getPackageDir(testDataPackage)
+except lsst.pex.exceptions.NotFoundError as err:
+    testDataDirectory = None
+
+
+TESTDIR = os.path.dirname(__file__)
+
+
+skipIfNoData = unittest.skipIf(testDataDirectory is None, "testdata_subaru must be set up")
```

Comment:
Since none of the tests run if data are not available it might be less repetitive to put the skip in `setUpClass` or `setUp`.

Reply:
Ah, I didn't realize the skip decorators worked on classes, too.  Will do.

Reply:
I haven't looked deeply but skipping at the class level vs during `setUp` *might* result in different skip statistics. I don't remember if there is a difference.